src/pytexbot/main.py

```diff
@@ -48,6 +48,11 @@ def __init__(self, *, intents: discord.Intents):
         super().__init__(intents=intents)
 
         self.tree = discord.app_commands.CommandTree(self)
+        self.attendee_emails = []
+        self.base_pretix_api_url = (
+            'https://pretix.eu/api/v1/organizers/pytexas/events/''2024/orders/'
```

Comment:
```suggestion
            'https://pretix.eu/api/v1/organizers/pytexas/events/2024/orders/'
```
Were these empty quotes necessary? Seems like it just concatenated them.